src/app/presentation/arrows/arrows.component.ts

```diff
@@ -1,5 +1,5 @@
-import {Component, OnInit} from '@angular/core';
-import {PresentationComponent} from '../presentation/presentation.component';
+import { Component, OnInit } from '@angular/core';
```

Comment:
We'll need to figure out formatting settings, looks like intellij does the opposite with spacing in imports